specification/mysql/resource-manager/Microsoft.DBforMySQL/preview/2017-12-01-preview/examples/ReplicasListByServer.json

```diff
@@ -0,0 +1,106 @@
+{
+  "parameters": {
+    "serverName": "testprimary",
+    "resourceGroupName": "TestGroup",
+    "api-version": "2017-12-01-preview",
+    "subscriptionId": "ffffffff-ffff-ffff-ffff-ffffffffffff"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "id": "/subscriptions/ffffffff-ffff-ffff-ffff-ffffffffffff/resourceGroups/TestGroup/providers/Microsoft.DBforMySQL/servers/testserver",
+            "name": "testserver",
+            "type": "Microsoft.DBforMySQL/servers",
+            "location": "onebox",
```

Comment:
"onebox" is not a customer region. better use a valid region for examples. I'm going through other examples as well to have this corrected.